build-tools/src/main/kotlin/org/jetbrains/kotlin/XcRunRuntimeUtils.kt

```diff
@@ -9,9 +9,9 @@ import kotlin.math.min
  * Compares two strings assuming that both are representing numeric version strings.
  * Examples of numeric version strings: "12.4.1.2", "9", "0.5".
  */
-private fun compareStringsAsVersions(version1: String, version2: String): Int {
-    val version1 = version1.split('.').map { it.toInt() }
-    val version2 = version2.split('.').map { it.toInt() }
+private fun compareStringsAsVersions(version1String: String, version2String: String): Int {
```

Comment:
It's obvious to caller that `version1String` has String type, so "String" suffix is meaningless. If you want to avoid name shadowing, it is better to rename local variable.